docs/benchmarks_tutorial.rst

```diff
@@ -0,0 +1,42 @@
+Benchmarks
+==========
+
+``petastorm-throughput.py`` command line tool makes it possible to measure :class:`~petastorm.reader.Reader` samples
+throughtput for a given dataset. The command line tool has several optional command line arguments, such as number of
+threads being used by the :class:`~petastorm.reader.Reader`, whether to use `Tensorflow` way of reading data
+(:func:`~petastorm.tf_utils.tf_tensors`) or pure Python one.
+
+As a toy example, let's generate a small `Hello World` dataset (by default the dataset is generated here:
+``/tmp/hello_world_dataset``):
+
+.. code-block:: bash
+
+ $ python examples/hello_world/generate_hello_world_dataset.py
+
+Now, let's run ``petastorm-throughput``:
+
+.. code-block:: bash
+
+ $ petastorm-throughput.py file:///tmp/hello_world_dataset
+ Average sample read rate: 709.84 samples/sec; RAM 217.12 MB (rss); CPU 136.30%
+
+Typically, you would want to find an optimal throughput by varying the following parameters:
+
+ - ``-w``: number of workers (controls number of loaders in the experimental :class:`~petastorm.reader.reader_v2.ReaderV2`).
```

Comment:
I think its a bit unclear the difference between number of loaders and thread based parallelism here

Reply:
Clarified a little. Once we get rid of V1, we'll cleanup the documentation.